bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -65,6 +62,10 @@
 from bosdyn.util import seconds_to_duration
 from geometry_msgs.msg import Pose, TransformStamped
 from google.protobuf import wrappers_pb2
+from perception_and_utils.utils.conversions import (
```

Comment:
Does this utils require the new package installed? If it does, could we somehow update the install file or use something like try and except to see users have that installed since I do not want to break the existing code

Reply:
This is a new package so it would need installation.

```
cd spot_sim2real/
cd perception_and_utils_root/
pip install -e . && cd ../
```
These steps are needed. Once user does that, it should be all fine.